src/types.ts

```diff
@@ -33,6 +33,15 @@ export interface Instance {
     bid: Bid,
     signer: ethers.Signer
   ) => Promise<ethers.ContractTransaction>;
+  buyNow: (
+    signer: ethers.Signer,
```

Comment:
The other functions on this interface order the parameters as `params, signer` but this one does `signer, params`. Can you change these to follow the pattern of `params..., signer` (signer is at the end)